redux/redux.d.ts

```diff
@@ -1,50 +1,79 @@
-// Type definitions for Redux v1.0.0
+// Type definitions for Redux v3.0.0
 // Project: https://github.com/rackt/redux
-// Definitions by: William Buchwalter <https://github.com/wbuchwalter/>, Vincent Prouillet <https://github.com/Keats/>
+// Definitions by: William Buchwalter <https://github.com/wbuchwalter/>,
+//                 Vincent Prouillet <https://github.com/Keats/>,
+//                 Michael Tiller <https://github.com/xogeny/>
 // Definitions: https://github.com/borisyankov/DefinitelyTyped
 
 declare module Redux {
+    // Minimalist Action definition
+    interface Action {
+        type: string;
+    }
 
+    // Not used...?
     interface ActionCreator extends Function {
-        (...args: any[]): any;
+        (...args: any[]): Action;
```

Comment:
It doesn't seems complete according to the glossary : http://redux.js.org/docs/Glossary.html#action-creator
```
(...args: any[]): Action|AsyncAction;
```